sources/netdog/Cargo.toml

```diff
@@ -8,11 +8,16 @@ publish = false
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
+default = []
+wicked = []
+
 [dependencies]
+anyhow = "1.0"
 argh = "0.1"
 dns-lookup = "2"
 ipnet = { version = "2", features = ["serde"] }
-imdsclient = { path = "../../imdsclient", version = "0.1" }
+imdsclient = { path = "../imdsclient", version = "0.1" }
```

Comment:
There's another source of conditional compilation implied by this - `imdsclient` is used in `generate_hostname.rs` which has these:
```
#[cfg(variant_platform = "aws")]
#[cfg(not(variant_platform = "aws"))]
```